.github/workflows/pre-release.yml

```
@@ -0,0 +1,100 @@
+          python-version: '3.10'
+          cache: 'pip'
+
+      - name: Install Python dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -e .
+          pip install build twine pyyaml tomli
+
+      - name: Get version from pyproject.toml
+        id: version
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:2:28: Double quote to prevent globbing and word splitting

(shellcheck)